### PR TITLE
Adds accessibility testing

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds accessibility testing.

## Additions

- Adds `gulp test:a11y` accessibility testing using node-wcag.

## Changes

- Adds binaryDirectory parameter to fsHelper.getBinary helper function.

## Testing

- See bottom of `TEST.md`

## Review

- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim 

## Todos

- **NOTE: The change noted in https://github.com/cfpb/node-wcag/issues/35 needs to be fixed before merging this PR or this will only work with the homepage.**

### CODE REVIEW

File: gulp/tasks/test.js [hunk 1]
```diff
@@ -95,6 +94,18 @@ function _getProtractorParams() {
   return params;
 }
 
+function _getWCAGParams() {
```
Comment (sebworks) [inline gulp/tasks/test.js]: Should this function have JSDoc comments?

**Comment by sebworks** [comment]
A couple or errors when running `gulp lint`.

File: gulp/tasks/test.js [hunk 2]
```diff
@@ -95,6 +94,18 @@ function _getProtractorParams() {
...
+  var commandLineParams = minimist( process.argv.slice( 2 ) );
+  var host = process.env.HTTP_HOST || 'localhost';
+  var port = process.env.HTTP_PORT || '7000';
```
Comment (jimmynotjim) [inline gulp/tasks/test.js]: This port should now be `8000`

File: gulp/tasks/test.js [hunk 3]
```diff
@@ -111,6 +122,17 @@ gulp.task( 'test:coveralls', function() {
     .pipe( $.coveralls() );
 } );
 
+// This task will only run on Travis
+gulp.task( 'test:a11y', function() {
+  spawn(
+    fsHelper.getBinary( 'wcag', '.bin' ),
+    _getWCAGParams(),
+    { stdio: 'inherit' } )
```
Comment (jimmynotjim) [inline gulp/tasks/test.js]: Can we break the closing parens to a new line matching indent of `spawn` so that it's clear it's closing that function. This block is kind of hard to follow.

File: gulp/tasks/test.js (hunk 1 above)
Comment (anselmbradford) [inline gulp/tasks/test.js]: Yes! Added in https://github.com/cfpb/cfgov-refresh/commit/4550c8f0ea9e3a46e07fa875445a04de35a54205

File: gulp/tasks/test.js (hunk 2 above)
Comment (anselmbradford) [inline gulp/tasks/test.js]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/2ec4d5b08584b32f28420a5429edda0eb067a77e

File: gulp/tasks/test.js (hunk 3 above)
Comment (anselmbradford) [inline gulp/tasks/test.js]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/2ec4d5b08584b32f28420a5429edda0eb067a77e

**Comment by anselmbradford** [comment]
This is ready to review again cc @sebworks @jimmynotjim @KimberlyMunoz

**Comment by jimmynotjim** [comment]
Runs well locally, :+1:

**Comment by sebworks** [comment]
:+1:
